samples/blog/css/algolia.css

```diff
@@ -1,3 +1,20 @@
+input[type="text"].algolia-magic-search-box--input,
+.algolia-magic-search-box--input {
```

Comment:
Nope, but this is used to override another selector applied on every `input[type="text"]` (which is stronger than a simple `.class`). I think this is Bootstrap not playing nicely here, but not sure.